src/app/address/googleAutocomplete/GoogleAutocompleteFormField.tsx

```diff
@@ -76,7 +78,7 @@ const GoogleAutocompleteFormField: FunctionComponent<GoogleAutocompleteFormField
         <div className={ `dynamic-form-field dynamic-form-field--addressLineAutocomplete` }>
             <FormField
                 input={ renderInput }
-                labelContent={ labelContent }
+                labelContent={ null }
```

Comment:
I see you are lifting the label to the component where autocomplete is rendered? But why don't we pass the label inside here so `<FormField` is updated accordingly?

Reply:
Think in that case you can get rid of the above line completely, it's an optional prop

Reply:
I have removed this line.